src/Zusi3TCP.h

```diff
@@ -26,6 +26,10 @@ SOFTWARE.
 #include <cstdint>
 #include <vector>
 #include <set>
+#include <string>
+#include <exception>
+
+#include "string.h" // For memcpy
```

Comment:
Is there a reason why you decided to write it like this instead of #include <cstring>?